src/main/java/com/icthh/xm/ms/entity/domain/FunctionResultContext.java

```diff
@@ -1,7 +1,7 @@
 package com.icthh.xm.ms.entity.domain;
 
 import com.icthh.xm.commons.domain.FunctionResult;
-import com.icthh.xm.commons.logging.util.MdcUtils;
+import org.springframework.cloud.cloudfoundry.com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
wrong package, can we use directly `com.fasterxml.jackson.annotation.JsonIgnore`

Reply:
done